### PR TITLE
ref(AM): Change names of `rps` and `rpm` functions

### PR DESCRIPTION
**Pull request by lobsterkatie**
Updating the docs to reflect the changes made [here](https://github.com/getsentry/sentry/pull/18917) and [here](https://github.com/getsentry/sentry/pull/19007).

(I also fixed a typo that's been bothering me.)

### CODE REVIEW

File: src/collections/_documentation/performance/distributed-tracing.md [hunk 1]
```diff
@@ -297,8 +297,8 @@ The following functions aggregate transaction counts and the rate at which trans
 
 - count
 - count unique values (for a given field)
-- average requests (transactions) per second
-- average requests (transactions) per minute
+- average events (transactions) per second (eps)
+- average event (transactions) per minute (epm)
```
Comment (rhcarvalho) [inline src/collections/_documentation/performance/distributed-tracing.md:301]: Did you mean 

```suggestion
- average events (transactions) per minute (epm)
```

File: src/collections/_documentation/performance/performance-metrics.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ The P95 Threshold indicates that 5% of transaction durations are greater than th
 ## P99 Threshold
 The P99 Threshold indicates that 1% of transaction durations are greater than the threshold. For example, if the P99 threshold is 5 seconds, then 1% of transactions exceeded that threshold, taking longer than 5 seconds.
 
-## Throughput (Total, RPM, RPS)
-Throughput indicates the number of requests over a given time range (Total), requests per minute (RPM), or requests per second (RPS).
+## Throughput (Total, TPM, TPS)
+Throughput indicates the number of transactions over a given time range (Total), average transactions per minute (EPM), or average transactions per second (EPS).
```
Comment (evanh) [inline src/collections/_documentation/performance/performance-metrics.md:31]:
```suggestion
Throughput indicates the number of transactions over a given time range (Total), average transactions per minute (TPM), or average transactions per second (TPS).
```

File: src/collections/_documentation/performance/distributed-tracing.md (hunk 1 above)
Comment (lobsterkatie) [reply src/collections/_documentation/performance/distributed-tracing.md:301]: D'oh! Good catch, thanks.

File: src/collections/_documentation/performance/performance-metrics.md (hunk 2 above)
Comment (lobsterkatie) [reply src/collections/_documentation/performance/performance-metrics.md:31]: Dammit - caught this on Friday and then forgot to push. 🤦🏻‍♀️

Thank you!

**Review by evanh — APPROVED** [review]

**Pull request merged**